terraform/modules/nitro/main.tf

```
@@ -0,0 +1,38 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+resource "aws_instance" "nitro" {
+  ami               = "ami-0dfcb1ef8550277af"
```

You can filter for the most recent amazon linux 2 ami. Normally I would pin a version we like, but since this whole repo seems to be for temporary deployments, it could make your life easier to grab the latest AMI

```suggestion
data "aws_ami" "latest_amazon_linux_2" {
  owners = ["amazon"]
  # Note, this data source represents a single AMI, so we sort by most recent to
  # guarantee a single result.
  most_recent = true

  filter {
    name = "name"
    # filter for standard Amazon Linux 2 AMIs using HVM virtualization, x86_64
    # architecture, and EBS General Purpose SSD (gp2) as the root device.
    values = ["amzn2-ami-hvm-*-x86_64-gp2"]
  }
}

resource "aws_instance" "nitro" {
  ami             = data.aws_ami.latest_amazon_linux_2.id
```

---

using the pin version for what we know works.  We can update and add more dynamic functionality as we learn more.